routes/teams.js

```diff
@@ -23,4 +24,101 @@ router.get('/team', function(req, res) {
     });
 });
 
+
+// when players endpoint is hit, call the API/DB using that team_id
+router.get('/players', function(req, res) {
+  temp = req.query.split('=');
+  team_id = temp[1];
+  console.log("team id is: "+team_id);
+  fetchPlayers(team_id)
```

Comment:
it looks like you are missing the final piece to return the json response. it should be something like ```players = fetchPlayers(team_id);     	res.json(players);```